bgpd/bgp_vty.c

```diff
@@ -2332,6 +2332,36 @@ DEFUN (no_bgp_timers,
 	return CMD_SUCCESS;
 }
 
+/* BGP minimum holdtime.  */
+
+DEFUN(bgp_minimum_holdtime, bgp_minimum_holdtime_cmd,
+      "minimum-holdtime (0-65535)",
```

Comment:
Please, put this under "bgp minimum-holdtime...".

Reply:
Fixed in b042667a3da442d086f1e14b30936f374e9374fd